tests/git_push_test.go

```diff
@@ -128,6 +128,13 @@ var _ = Describe("git push deis master", func() {
 
 					})
 
+					Specify("and can execute deis run successfully", func() {
+						git.Push(user, keyPath, app, "Powered by Deis")
+						sess, err := cmd.Start("deis run env -a %s", &user, app.Name)
+						Expect(err).NotTo(HaveOccurred())
+						Eventually(sess).Should(Exit(0))
```

Comment:
Not really, we don't care about the output, at all, just that it runs successfully